### PR TITLE
Remove autoprefixer since it's included with postcss

### PR DESCRIPTION
**Pull request by JaKXz**
Connected to rangle/rangle-starter#57

can @SethDavenport review?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,8 +37,6 @@
   },
   "devDependencies": {
     "angular2": "2.0.0-beta.9",
-    "autoprefixer": "^6.3.3",
```
Comment (SethDavenport) [inline package.json]: I'm pretty sure we still need these packages for npm 2 (since it doens't flatten the tree like npm 3 does).

Comment (JaKXz) [inline package.json]: it's a peer dependency of `postcss`? That's unfortunate...

Comment (SethDavenport) [inline package.json]: Of postcss-next.

```sh
nvm use v4
rm -rf node_modules
npm install
npm run dev
```

```sh
ERROR in ./src/styles/index.css
Module build failed: Error: Cannot find module 'autoprefixer'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.module.exports.postcss (/Users/sdavenpo/code/rangle/angular2-redux-starter/webpack.config.js:104:7)
    at Object.module.exports (/Users/sdavenpo/code/rangle/angular2-redux-starter/node_modules/postcss-loader/index.js:28:27)
 @ ./src/containers/app.ts 50:21-51
...
```

Comment (JaKXz) [inline package.json]: The reason I'm confused is that `npm run build` works on CI.

Comment (SethDavenport) [inline package.json]: CI doesn't execute `npm run dev`, which is where the error happens.

**Pull request merged**